packages/n/niftiheader/xmake.lua

```diff
@@ -0,0 +1,14 @@
+package("niftiheader")
+    set_kind("library", {headeronly = true})
+    set_homepage("https://nifti.nimh.nih.gov/")
```

Comment:
please add `set_description` and `set_license`

Reply:
and add a new line after `set_license`, you can refer other packages in xmake-repo.